corehq/apps/users/templates/users/domain_permissions_mirror.html

```diff
@@ -18,26 +18,31 @@
   </p>
 
  <form class="form form-horizontal hq-form"
-      name="create_new_mirror"
-      action="{% url "create_new_mirror" domain %}"
-      method="POST">
+       name="create_new_mirror"
+       action="{% url 'create_new_mirror' domain %}"
+       method="POST">
   {% csrf_token %}
   <input type="hidden" name="form_type" value="create_new_mirror"/>
-  <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#myModal"></i> {% trans "Create New Mirror for Domain" %}</button>
-  <div id="myModal" class="modal fade" role="dialog" style="background: #fff">
-    <div class="modal-dialog" style="width:600px margin:30px auto">
-      <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal">&times;</button>
-        <h3 class="modal-title">Create Mirror</h3>
-      </div>
-      <div class="modal-body">
-        <label for="mirror_domain">{% trans "Mirror: " %}</label>
-        <input type="text" id="mirror_domain" name="mirror_domain" value="{{ mirror_domain }}"/>
-
-      </div>
-      <div class="modal-footer">
-        <a href="#" data-dismiss="modal" class="btn btn-default">{% trans "Cancel" %}</a>
-        <button type="submit" class="btn btn-primary"> {% trans "Create Mirror" %}</button>
+  <button type="button"
+          class="btn btn-primary"
+          data-toggle="modal"
+          data-target="#myModal">
+    <i class="fa fa-plus"></i> {% trans "Create New Domain Permission Mirror" %}</button>
+  <div id="myModal" class="modal fade" role="dialog">
+    <div class="modal-dialog" style="width:600px; margin:30px auto">
+      <div class="modal-content rounded-0" style="background: #fff">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+          <h3 class="modal-title">{% trans "Create a new domain permission mirror" %}</h3>
+        </div>
+        <div class="modal-body">
+          <label for="mirror_domain" style="padding:8px">{% trans "Mirror: " %}</label>
```

Comment:
There are additional classes that work with `form-horizontal` to create the kind of form you're making, with a column of labels on the left and column of inputs on the right. Check out Bootstrap's [horizontal-form docs](https://getbootstrap.com/docs/3.3/css/#forms-horizontal).

Alternatively, you could use [crispy forms](https://django-crispy-forms.readthedocs.io/en/latest/), where you set up a python object and then django renders the HTML for you. I don't love crispy forms because I think they're often more complexity than they're worth, but this is a small form, so if you like that approach, this is a fine place to use it. If you want to try out crispy, the `CopyApplicationForm` is a small one to look at as an example, and [here](https://github.com/dimagi/commcare-hq/blob/a0f4e4bcbd6cdbe8d45918d2eeaf478300a996c8/corehq/apps/app_manager/forms.py#L53-L65) is the code that sets up the UI.

Whichever approach you choose, check out the [style guide section on forms](https://www.commcarehq.org/styleguide/organisms/#organisms-forms) to make sure what you're doing is consistent with other forms in HQ.